packages/@sanity/field/src/diff/components/DiffCard.tsx

```diff
@@ -23,6 +22,56 @@ interface DiffCardWithAnnotationProps {
   tooltip?: {description?: React.ReactNode} | boolean
 }
 
+const StyledCard = styled(Card)`
+  --diffcard-radius: ${({theme}) => rem(theme.sanity.radius[2])};
```

Comment:
How does this relate to the RoundedCard component above?

Reply:
This is specifically for the div around the change title segment inside the diff card 👍